lib/wallet/http.js

```diff
@@ -1285,43 +1285,47 @@ class HTTP extends Server {
 
       const options = TransactionOptions.fromValidator(valid);
 
-      // if already exists in the cache do not process it
-      let results = [];
-      let errors  = [];
+      const uniqueBids = [];
+      const bidResults = [];
 
-      const filteredBids = bids.filter((bid) => {
-          const bidResultFromCache = req.wallet
-          .sendBatchBidResults.get(bid.idempotencyKey);
+      let errors  = [];
 
-          if (bidResultFromCache != null) {
-              bidResultFromCache['fromCache'] = true;
-              results.push(bidResultFromCache);
-          }
+      const bidCache = req.wallet.sendBidResults;
 
-          return (bidResultFromCache == null);
-      });
+      for (const bid of bids) {
+        const bidResultFromCache = bidCache.get(bid.idempotencyKey);
+        bidResultFromCache ? bidResults.push(bidResultFromCache)
```

Comment:
style nit: would prefer to only use ternaries in value positions and `if/else` statements for control flow

Reply:
fixed ... changed to if else